### PR TITLE
388 human readable formats

### LINKED ISSUE #388: Support human readable sizes in config
https://crates.io/crates/ubyte

### PR DESCRIPTION
**Pull request by Boneyan**
Configs now support human readable formats (i.e. kB, KiB) in max_blob_size & bloom_filter_memory_limit
Closes #388

### CODE REVIEW

File: bob-common/src/configs/node.rs [hunk 1]
```diff
@@ -330,12 +332,12 @@ impl Pearl {
         self.max_data_in_blob
     }
 
-    fn default_max_blob_size() -> u64 {
-        1_000_000
+    fn default_max_blob_size() -> ByteUnit {
+        ByteUnit::Byte(1_000_000)
```
Comment (vovac12) [inline bob-common/src/configs/node.rs:336]: You can use `ByteUnit::MB`

Comment (Boneyan) [reply bob-common/src/configs/node.rs:336]: And it would be better?

Comment (vovac12) [reply bob-common/src/configs/node.rs:336]: Yeah

**Comment by ikopylov** [comment]
Please add a description of supported size formats to the header of `config-examples/node.yaml` (https://github.com/qoollo/bob/blob/master/config-examples/node.yaml) in the same way as it is described for time formats

**Comment by ikopylov** [comment]
LGTM

File: config-examples/node.yaml [hunk 2]
```diff
@@ -36,17 +53,17 @@ open_blobs_hard_limit: 10
 http_api_port: 8000
 # http REST api address
 http_api_address: 0.0.0.0
-# memory limit for all bloom filters. Unlimited if not specified
-bloom_filter_memory_limit: 8589934592
+# [size] memory limit for all bloom filters. Unlimited if not specified
+bloom_filter_memory_limit: 8589934592 b
```
Comment (ikopylov) [inline config-examples/node.yaml:57]: Please, make this value esier to read by converting to Gb:
```suggestion
bloom_filter_memory_limit: 8 gb
```

File: config-examples/node.yaml [hunk 3]
```diff
@@ -36,17 +53,17 @@ open_blobs_hard_limit: 10
...
 # used only for 'backend_type: pearl'
 pearl:
   # optional, default = false, enables linux AIO
   enable_aio: false
   # path to logfile with info about disks states switches, default = "/tmp/bob_events.csv"
   disks_events_logfile: "/tmp/bob_events.csv"
-  # size in bytes. required for 'pearl'
-  max_blob_size: 1000000
+  # [size] required for 'pearl'
+  max_blob_size: 1000000 b
```
Comment (ikopylov) [inline config-examples/node.yaml:66]:
```suggestion
  max_blob_size: 100 mb
```

File: config-examples/node.yaml [hunk 4]
```diff
@@ -36,17 +53,17 @@ open_blobs_hard_limit: 10
...
 # used only for 'backend_type: pearl'
 pearl:
   # optional, default = false, enables linux AIO
   enable_aio: false
   # path to logfile with info about disks states switches, default = "/tmp/bob_events.csv"
   disks_events_logfile: "/tmp/bob_events.csv"
-  # size in bytes. required for 'pearl'
-  max_blob_size: 1000000
+  # [size] required for 'pearl'
+  max_blob_size: 1 mb
```
Comment (ikopylov) [inline config-examples/node.yaml:66]:
```suggestion
  max_blob_size: 100 mb
```

**Pull request merged**